### PR TITLE
[Botskills] Unit Test compability for Linux

### PR DESCRIPTION
**Pull request by VictorGrycuk**
Related to #xxxx

### Purpose
*What is the context of this pull request? Why is it being done?*
Fix Botskills compatibility with Unix and updates several unit tests:
- Filenames were updated to avoid issues with case-sensitive directories
- Replace `\n`, `\r\n`, and hardcoded new lines with platform specific new lines
- Avoid using hardcoded directory separator
- Add support for forward slashes in `sanitizationUtils.ts`
- Improve how the luis file is generated in `connectSkill.ts`
- Update the tests to accommodate the above modifications
- Update the tests to use platform the platform specific error message

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*
The necessary tests were updated to accommodate these medications.

image

image

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [x] I have commented my code, particularly in hard-to-understand areas	
- [x] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

**Review by Batta32 — COMMENTED** [review]
The changes are ok, however, I have some questions:
1. I'd like to have a consistency in the use of all the `EOL` if it's possible. If not, it's ok!
1. Have you ran `eslint`?
1. I saw several times the use of `interpolation` and `+` in the same place. Is this ok?

**Comment by VictorGrycuk** [comment]
@Batta32 
1. I am not sure what you mean by consistency on the use of all the EOL. Are they not used the same way everywhere?
2. I haven't, but I can do that now.
3. I didn't see the point of closing an interpolation to immediately open it again, but, in the end it is the same. For example:
```
`Command: ${ luisConvertCommand.join(' ') }${ EOL }${ err }`
`Command: ${ luisConvertCommand.join(' ') + EOL + err }`
```
I am just replacing `}${` with a `+`

**Comment by Batta32** [comment]
1. I see that some `EOL` are used in the same line of the text but others not. Maybe because the text is too long? 🤔 
1. Ack
1. Ack, it's ok.

File: tools/botskills/test/childProcess.test.js [hunk 1]
```diff
@@ -8,21 +8,34 @@ const { ChildProcessUtils } = require("../lib/utils");
...
+            case 'win32':
+                return `'${ command }' is not recognized as an internal or external command,${
+                        EOL }operable program or batch file.${ EOL }`;
+            case 'linux':
+                return `/bin/sh: 1: ${ command }: not found${ EOL }`;
+            default:
+                return 'Unrecognized OS';
+        }
+      }
+
```
Comment (Batta32) [inline tools/botskills/test/childProcess.test.js:26]: This is weird here, but I understand it

**Review by Batta32 — CHANGES_REQUESTED** [review]
The EOL and `parse` changes might be better to add them in PR #455 , this should only contain things related to tests. What do you think?

**Review by FedericoBernal — APPROVED** [review]
LGTM

**Pull request merged**